compiler+runtime/test/jank/form/try/pass-nested-try-with-finally.jank

```diff
@@ -0,0 +1,14 @@
+;; Test nested try blocks with finally
```

Comment:
We already have tests for this. Why is this one needed?

Reply:
removed.